normandy/control/static/control/js/components/RecipeForm.jsx

```diff
@@ -30,19 +32,32 @@ export class RecipeForm extends React.Component {
     });
   }
 
+  validateForm(formValues) {
+    let jexlExpression = formValues.filter_expression;
+    return jexl.eval(jexlExpression, {});
```

Comment:
I belive that's so that lines 45 through 59 can handle the response from the jexl.eval 
@brittanystoroz is that right?